include/portfolio_solver.h

```diff
@@ -0,0 +1,61 @@
+
+#pragma once
+
+#include "smt_defs.h"
```

Comment:
This includes everything right now because of a bunch of copying and pasting. Will remove unnecessary includes.